config/locales/loa1/en.yml

```diff
@@ -0,0 +1,27 @@
+---
+en:
+  loa:
+    cancel:
+      modal_header: Are you sure you want to cancel?
+      warning_header: If you cancel now
+  loa1:
+    buttons:
+      continue: Continue account creation
+      cancel: Cancel and delete your information
+    cancel:
+      success: The record of your information has been cleared
+      warning_points:
+        - You won’t have a login.gov account
+        - We won't keep a record of your email address, password, and phone number
+        - You won't be able to access your benefits information with the Railroad Retirement Board
```

Comment:
Good point. Does this modal only apply to RRB? If so, then we should add logic such that it is only displayed if the user came from RRB. Otherwise, we'll need to provide a way to customize this particular line for each agency, or rewrite it such that it can apply to any agency.